### ISSUE TITLE
Add Roles Based Authentication/Permissions Rules to the Calendar

### ISSUE THREAD

**Issue by paul-rinaldi** [codeformilwaukee/DECARCERATION-PLATFORM#29]
**Goals**:
- A service provider should be able to add events to the Calendar.
- A user, not logged in, should not be able to add events to the calendar.

**This may involve:**
- AWS 
- if statements on the calendar page (or on the link to the calendar page)
- API calls to AWS for authentication

_Useful tips_: 
- We might want to add a roles.js to the constants folder. 

**Roles**:
- Public (not logged in)
- Service Provider (logged in)

_Note_ to keep in mind but don't need to implement for this card.
- Eventually (Admin (logged in - can do edit, delete, add directly on calendar))

**Comment by paul-rinaldi** [comment]
Authentication library to use: https://aws-amplify.github.io/docs/js/authentication

**Comment by hdenisenko** [comment]
@paul-rinaldi do we have any aws infrastructure now with the events for the calendar? I haven't worked with aws before, but have time to learn and would be happy to grab this ticket if we have all the pieces needed for it already!

**Comment by paul-rinaldi** [comment]
@hdenisenko yes, I'm working on migrating some old db for events on one account to another

**Comment by hdenisenko** [comment]
happy to pick this up when that's ready!
